core/src/main/java/org/polypheny/db/sql/ddl/altermaterializedview/SqlAlterMaterializedAddIndex.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2019-2021 The Polypheny Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.polypheny.db.sql.ddl.altermaterializedview;
+
+import static org.polypheny.db.util.Static.RESOURCE;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import org.polypheny.db.adapter.DataStore;
+import org.polypheny.db.catalog.Catalog.TableType;
+import org.polypheny.db.catalog.entity.CatalogTable;
+import org.polypheny.db.catalog.exceptions.GenericCatalogException;
+import org.polypheny.db.catalog.exceptions.UnknownColumnException;
+import org.polypheny.db.catalog.exceptions.UnknownDatabaseException;
+import org.polypheny.db.catalog.exceptions.UnknownKeyException;
+import org.polypheny.db.catalog.exceptions.UnknownSchemaException;
+import org.polypheny.db.catalog.exceptions.UnknownTableException;
+import org.polypheny.db.catalog.exceptions.UnknownUserException;
+import org.polypheny.db.ddl.DdlManager;
+import org.polypheny.db.ddl.exception.AlterSourceException;
+import org.polypheny.db.ddl.exception.IndexExistsException;
+import org.polypheny.db.ddl.exception.MissingColumnPlacementException;
+import org.polypheny.db.ddl.exception.UnknownIndexMethodException;
+import org.polypheny.db.jdbc.Context;
+import org.polypheny.db.sql.SqlIdentifier;
+import org.polypheny.db.sql.SqlNode;
+import org.polypheny.db.sql.SqlNodeList;
+import org.polypheny.db.sql.SqlUtil;
+import org.polypheny.db.sql.SqlWriter;
+import org.polypheny.db.sql.ddl.SqlAlterMaterializedView;
+import org.polypheny.db.sql.parser.SqlParserPos;
+import org.polypheny.db.transaction.Statement;
+import org.polypheny.db.transaction.TransactionException;
+import org.polypheny.db.util.ImmutableNullableList;
+
+public class SqlAlterMaterializedAddIndex extends SqlAlterMaterializedView {
```

Comment:
Please rename class to SqlAlterMaterialized**View**AddIndex

Reply:
Renamed to SqlAlterMaterializedViewAddIndex